database/migration/src/main/resources/changesets/sub_workflow_store.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.3.xsd">
+
+    <property name="clob.type" value="LONGTEXT" dbms="mysql"/>
+    <property name="clob.type" value="LONGVARCHAR" dbms="hsqldb"/>
+
+    <changeSet author="tjeandet" id="SUB_WORKFLOW_STORE_ENTRY">
```

Comment:
Oh ok, but then sub workflows would need to be added to the `WORKFLOW_STORE_ENTRY` table as well no ? That would be a problem I think considering how sub workflows are handled in the WEA.